postgres/init/21-portal-basedata.sql

```diff
@@ -124,6 +127,7 @@ COPY portal.company_statuses (id, label) FROM stdin;
 2	ACTIVE
 3	REJECTED
 4	INACTIVE
+5	DELETED
```

Comment:
changes to static data that is represented as enum must be done via code-change + migration. Otherwise the dotnet-code doesn't know about this change. Please remove from this PR and create a userstory for portal-backend, it will be very low effort.